index.html

```diff
@@ -14,262 +13,392 @@
+          <aside class="material">
+            <h1 class="see">
+              I’m Vivek Raj.<br />
+              Glad to see you!
+            </h1>
+            <p class="software">
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hesitate to contact me.
+            </p>
+            <h4 class="connect">LET'S CONNECT</h4>
+            <div class="media">
+              <ul class="social">
+                <li>
+                  <a href="https://twitter.com/BossVivekRaj">
+                    <img src="image/twitter.jpg" alt="twitter-icon" />
+                  </a>
+                </li>
+                <li>
+                  <a href="https://www.linkedin.com/in/vivek-raj-a78b92256/">
+                    <img src="image/Linkedin icon.jpg" alt="linkedin-icon" />
+                  </a>
+                </li>
+                <li>
+                  <a href="https://medium.com/">
+                    <img src="image/medium.jpg" alt="medium-icon" />
+                  </a>
+                </li>
+                <li>
+                  <a href="https://github.com/VivekRaaZ">
+                    <img src="image/github.jpg" alt="github-icon" />
+                  </a>
+                </li>
+                <li>
+                  <a href="https://www.angellist.com/">
+                    <img src="image/angellist.jpg" alt="angellist-icon" />
+                  </a>
+                </li>
+              </ul>
+            </div>
+          </aside>
+        </div>
+      </section>
+
+      <!-- ------------  GRID  ------------ -->
+      <div class="boss2">
+        <!-- ------------  Tonic  ------------ -->
+        <section class="tonic tonic4">
+          <aside class="desk">
+            <div class="nature">
+              <img
+                src="image/Snapshoot Portfoli.jpg"
+                alt="nature"
+                width="100%"
+                height="100%"
+              />
+            </div>
+          </aside>
+          <aside class="desk">
+            <div class="tonic2">
+              <h2 class="title">Tonic</h2>
+              <div class="canopy" id="raja">
+                <h6>CANOPY</h6>
+                <img
+                  src="image/Counter.jpg"
+                  alt="conter"
+                  width="8px"
+                  height="8px"
+                />
+                <p class="back">Back End Dev</p>
+                <img
+                  src="image/Counter.jpg"
+                  alt="conter"
+                  width="8px"
+                  height="8px"
+                />
+                <p class="back">2015</p>
+              </div>
+              <p class="daily">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
```

Comment:
Desktop design looks good so far. However, your mobile design is a bit different from the actual figma design. Kindly check and ensure that applying the desktop styling do not affect the mobile design. Below are a few changes I would like you to consider and fix: 

- [ ] The project cards of the mobile design should look be slightly different from what you have now. The picture should be on top of the text description and button as shown below. Some of the texts seem to be overflowing out of the project cards which needs to be fixed. 
- [ ] Unlike what you have now, the `about` section should have the description and social icons on top of the languages/frameworks/skills section.
- [ ] The contact form should have inputs with rounded corners, all inputs and textarea should have same width, and the submit button should be left aligned. The label of the submit button seems to be cut off as shown below.
Kindly check and fix the above issues and all other factors preventing your mobile version to match the design.

### Your designs  

![Screenshot from 2023-07-21 08-17-24](https://github.com/VivekRaaZ/portfolio/assets/65233860/e1277f18-0ec0-42a5-9475-a3b0a169d845) 

![Screenshot from 2023-07-21 08-17-38](https://github.com/VivekRaaZ/portfolio/assets/65233860/bec6385f-2885-49fb-9745-0bf1ac29969c)

![Screenshot from 2023-07-21 08-17-08](https://github.com/VivekRaaZ/portfolio/assets/65233860/1f36f638-6dec-4f18-bf1e-7bdeb4e64611)


![Screenshot from 2023-07-21 08-16-48](https://github.com/VivekRaaZ/portfolio/assets/65233860/97032eb3-08cc-4c36-9de2-493a2ba20d56)

### Actual designs

![Screenshot from 2023-07-21 08-22-15](https://github.com/VivekRaaZ/portfolio/assets/65233860/45de3bce-f34e-49df-acda-31635a7bd8bb)

![Screenshot from 2023-07-21 08-22-03](https://github.com/VivekRaaZ/portfolio/assets/65233860/c13805b5-35cc-4287-9011-4f8ac49d7430)

![Screenshot from 2023-07-21 08-22-41](https://github.com/VivekRaaZ/portfolio/assets/65233860/47f2a907-bd80-4a94-b71e-4606257b8df1)